apps/client/pages/post/add/index.page.tsx

```diff
@@ -26,11 +33,19 @@ const PostAdd = () => {
           {isModifyMode ? "Edit Post" : "New Post"}
         </Text>
       ),
+      backButtonCaution: true,
+      hideBackButton: isModifyMode ? undefined : true,
```

Comment:
hideBackButton의 atom 타입을 `hideBackButton?: boolean;`으로 변경하고 
```suggestion
      hideBackButton: !isModifyMode,
```

으로 사용하는 건 어떤가요? 제 생각에는 굳이 true라고 안해주고 살짝 느슨하고 boolean으로 해도 될 것 같습니다.

Reply:
아! 

hideBackButton?: boolean; 가 아닌 hideBackButton?: true;

로 굳이 한 이유가 있습니다.

hideBackButton?: boolean;

로 한다면 사용자는 true, false, undefined 라는 세가지 인터페이스에 마주하게 되고, true는 명확하지만 false와 undefined는 어떤 역할을 하는지 둘의 차이는 무엇인지 헷갈릴 수 있다고 생각했습니다.

hideBackButton의 취지에 맞게 true를 준다면!!!! backButton을 지우고, 그게 아니라면 undefined이라면! 지우지 않도록 하기 위함이었습니다. 이렇게 하면 사용자는 true를 주거나, 아예 안줘도 되기 때문에 인터페이스를 2개만 열어줬습니다!!

Reply:
오케이 확인하였습니다!